### PR TITLE
wikidata redirect match

### PR DESCRIPTION
**Pull request by alismayilov**
Redirect pattern is different in Wikidata from other Wiki pages. The pattern and matching therefore changed.

### CODE REVIEW

File: core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/wikipedia/Redirect.scala [hunk 1]
```diff
@@ -12,7 +12,7 @@ object Redirect
...
     private def mappings_redirects = Set("#REDIRECT")
     private def commons_redirects = Set("#REDIRECT")
-    private def wikidata_redirects = Set("#REDIRECT")
+    private def wikidata_redirects = Set("\"redirect\":")
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/wikipedia/Redirect.scala]: with this, I think we lose the redirects from pages that are not wikidata items but this is not important imo
(might worth making a comment on this though)

File: core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/simple/SimpleWikiParser.scala [hunk 2]
```diff
@@ -62,7 +62,14 @@ object SimpleWikiParser
       // - generating the XML dump files takes several days, and the wikitext is obviously not generated at the
       //   same time as the redirect target, so sometimes they do not match.
       // In a nutshell: if the redirect in WikiPage is different from what we find, we're probably correct.
+
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/simple/SimpleWikiParser.scala]: style nitpicking ;): 
```
val patttern = if (lang.wikiCode=="wikidata") Redirect(lang).mkString("")
  else """(?is)\s*(?:""" + Redirect(lang).mkString("|") + """)\s*:?\s*\[\[.*"""
```

**Comment by jimkont** [comment]
it's been a while since I looked at that part of the code but how / where do we parse the wikidata redirect item?

File: core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/wikipedia/Redirect.scala (hunk 1 above)
Comment (alismayilov) [reply core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/wikipedia/Redirect.scala]: Why do you think that we will lose redirects from pages that are not Wikidata items?

**Comment by JJ-Author** [comment]
@jimkont @alismayilov  what is the state here? I would like to either close this pr or merge it

**Pull request merged**